packages/cozy-harvest-lib/src/services/bi-http.js

```diff
@@ -73,6 +73,20 @@ const encodeToForm = rawForm => {
   return formData
 }
 
+export const getBIUserConfig = async (config, biAccessToken) => {
+  return await biRequest('GET', '/users/me/config', config, null, biAccessToken)
+}
+
+export const updateBIUserConfig = async (config, userConfig, biAccessToken) => {
```

Comment:
nit: having the same arg order than getBIUserConfig can be great (config, biAccessToken, userConfig) ?

Reply:
Actually, the order is consistent, it's just that the biAccessToken is always last. I am not very happy with it either... I keep you advice in mind for next time.